[2023-05-17T15:05:47.009+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: my_third_dagV5.fo manual__2023-05-17T15:05:45.880855+00:00 [queued]>
[2023-05-17T15:05:47.023+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: my_third_dagV5.fo manual__2023-05-17T15:05:45.880855+00:00 [queued]>
[2023-05-17T15:05:47.023+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-05-17T15:05:47.023+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-05-17T15:05:47.023+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-05-17T15:05:47.055+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): fo> on 2023-05-17 15:05:45.880855+00:00
[2023-05-17T15:05:47.065+0000] {standard_task_runner.py:55} INFO - Started process 14669 to run task
[2023-05-17T15:05:47.068+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'my_third_dagV5', 'fo', 'manual__2023-05-17T15:05:45.880855+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/dag3.py', '--cfg-path', '/tmp/tmpjqb3m5rj']
[2023-05-17T15:05:47.068+0000] {standard_task_runner.py:83} INFO - Job 175: Subtask fo
[2023-05-17T15:05:47.146+0000] {task_command.py:388} INFO - Running <TaskInstance: my_third_dagV5.fo manual__2023-05-17T15:05:45.880855+00:00 [running]> on host 031cfb56e8fb
[2023-05-17T15:05:47.226+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_third_dagV5
AIRFLOW_CTX_TASK_ID=fo
AIRFLOW_CTX_EXECUTION_DATE=2023-05-17T15:05:45.880855+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-17T15:05:45.880855+00:00
[2023-05-17T15:06:01.050+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/pyspark/context.py:317: FutureWarning: Python 3.7 support is deprecated in Spark 3.4.
  warnings.warn("Python 3.7 support is deprecated in Spark 3.4.", FutureWarning)

[2023-05-17T15:06:01.290+0000] {logging_mixin.py:137} INFO - Hello
[2023-05-17T15:06:01.290+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag3.py", line 70, in foo1
    print(spark.getConf().getAll())
AttributeError: 'SparkSession' object has no attribute 'getConf'
[2023-05-17T15:06:01.315+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=my_third_dagV5, task_id=fo, execution_date=20230517T150545, start_date=20230517T150547, end_date=20230517T150601
[2023-05-17T15:06:01.402+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 175 for task fo ('SparkSession' object has no attribute 'getConf'; 14669)
[2023-05-17T15:06:01.426+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-05-17T15:06:01.443+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
